db/migrations/postgres/000094_add_bio_to_users.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users ADD COLUMN IF NOT EXISTS bio VARCHAR(320) NOT NULL DEFAULT '';
+
+CREATE INDEX IF NOT EXISTS idx_users_bio_txt ON users USING gin(to_tsvector('english', bio));
```

Comment:
```suggestion
CREATE INDEX IF NOT EXISTS idx_users_bio_txt ON users USING gin(to_tsvector(bio));
```

According to the [PostgreSQL full text docs](https://www.postgresql.org/docs/15/textsearch-tables.html) we can omit the `'english'` parameter and...

> [the] query will use the configuration set by [default_text_search_config](https://www.postgresql.org/docs/15/runtime-config-client.html#GUC-DEFAULT-TEXT-SEARCH-CONFIG).